test/system/testBalances.js

```diff
@@ -1,4 +1,4 @@
-const runScenario3 = (trader2) => {
+const TestBalances = (trader1, trader2) => {
```

Comment:
Good idea - this will make locating particular scenario code easier.
Might be worth moving this (and the other scenarios) into a directory called 'scenarios/'